runtime/doc/api.txt

```diff
@@ -2685,6 +2685,13 @@ nvim_buf_set_extmark({buffer}, {ns_id}, {line}, {col}, {opts})
                     • "overlay": display over the specified column, without
                       shifting the underlying text.
                     • "right_align": display right aligned in the window.
+                    • "eol_right_align": display right aligned in the window
```

Comment:
keys should be alphabetical. and for this PR at least, "eol_right_align" should go after "eol".